themes/custom/az_barrio/az_barrio.theme

```diff
@@ -48,9 +55,38 @@ function az_barrio_theme_suggestions_input_alter(&$suggestions, $variables) {
 }
 
 /**
- * Implements hook_preprocess_form_element().
+ * Implements hook_preprocess_HOOK() for form elements.
  */
 function az_barrio_preprocess_form_element(array &$variables) {
+  if (isset($variables['label']['#title'])) {
+    if (is_object($variables['label']['#title']) && !$variables['label']['#title'] instanceof MarkupInterface) {
+      $variables['input_title'] = NULL;
+    }
+    elseif (is_object($variables['label']['#title'])) {
+      $variables['input_title'] = $variables['label']['#title'];
+    }
+    else {
+      $variables['input_title'] = $variables['label']['#title'];
+    }
```

Comment:
If the assignments here are correct, then it looks like the elseif is not needed:

```suggestion
    if (is_object($variables['label']['#title']) && !$variables['label']['#title'] instanceof MarkupInterface) {
      $variables['input_title'] = NULL;
    }
    else {
      $variables['input_title'] = $variables['label']['#title'];
    }
```

Reply:
Good catch.  Thanks @bberndt-uaz!